cmd/nvidia_gpu/nvidia_gpu.go

```diff
@@ -87,7 +89,7 @@ func main() {
 		}
 	}
 	glog.Infof("Using gpu config: %v", gpuConfig)
-	ngm := gpumanager.NewNvidiaGPUManager(devDirectory, mountPaths, gpuConfig)
+	ngm := gpumanager.NewNvidiaGPUManager(devDirectory, procDirectory, mountPaths, gpuConfig)
```

Comment:
I changed procDirectory to be an input when initializing NvidiaGPUManager, so that the unit test can directly replace this procDirectory with a test path. Changing the procDirectory to be an input doesn't change the existing logic, and it helps to save effort to write a set of mock functions for overriding (otherwise, the procDirectory is unchangeable).